src/scripts/install.sh

```diff
@@ -1,101 +1,35 @@
 # shellcheck disable=SC2148
-ORB_STR_AWS_CLI_VERSION="$(circleci env subst "${ORB_STR_AWS_CLI_VERSION}")"
-ORB_EVAL_INSTALL_DIR="$(eval echo "${ORB_EVAL_INSTALL_DIR}")"
-ORB_EVAL_BINARY_DIR="$(eval echo "${ORB_EVAL_BINARY_DIR}")"
+AWS_CLI_STR_AWS_CLI_VERSION="$(echo "${AWS_CLI_STR_AWS_CLI_VERSION}" | circleci env subst)"
+AWS_CLI_EVAL_INSTALL_DIR="$(eval echo "${AWS_CLI_EVAL_INSTALL_DIR}" | circleci env subst)"
+AWS_CLI_EVAL_BINARY_DIR="$(eval echo "${AWS_CLI_EVAL_BINARY_DIR}" | circleci env subst)"
 
-if grep "Alpine" /etc/issue >/dev/null 2>&1; then
+eval "$SCRIPT_UTILS"
+detect_os
+
+if [ "$SYS_ENV_PLATFORM" = "linux_alpine" ]; then
     if [ "$ID" = 0 ]; then export SUDO=""; else export SUDO="sudo"; fi
 else
     if [[ $EUID == 0 ]]; then export SUDO=""; else export SUDO="sudo"; fi
 fi
```

Comment:
Maybe extract this to `utils.sh` under a `set_sudo` function or something like that?